tests/components/Forms/JobSearchForm.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent, waitFor} from '@testing-library/react'
+import Form from '../../../src/components/Jobs/Form/Form'
+
+
+window.fetch = jest.fn().mockImplementation((submit, page) => {
+  return Promise.resolve({ submit, page });
+});
+
+describe("Form", () => {
```

Comment:
Please update this description block with the meaningful name for this form once moved to the Forms directory.

Reply:
done